postpic/io/__init__.py

```diff
@@ -21,25 +21,41 @@
 The postpic.io module provides free functions for importing and exporting data.
 '''
 
-from .csv import _export_field_csv
+from .csv import _export_field_csv, _import_field_csv
 from .npy import _export_field_npy, _import_field_npy
 from .vtk import export_scalar_vtk, export_vector_vtk, export_scalars_vtk
+from .image import _import_field_image, _import_field_image_extensions
 
 __all__ = ['export_field', 'load_field',
            'export_scalar_vtk', 'export_scalars_vtk', 'export_vector_vtk']
 
 
 def load_field(filename):
     '''
-    construct a new field object from file. currently, the following file
-    formats are supported:
-    *.npz
+    Load a field object previously stored using the `saveto` method. These are .npz files
+    with a specific metadata layout.
     '''
-    if not filename.endswith('npz'):
+    if not filename.lower().endswith('npz'):
         raise ValueError('File format of filename {0} not recognized.'.format(filename))
     return _import_field_npy(filename)
 
 
+def import_field(filename, **kwargs):
+    '''
+    Construct a new field object from foreign data. Currently, the following file
+    formats are supported:
+    *.png
+    *.tif
+    *.csv
+    '''
+    if filename.lower().endswith('csv'):
+        return _import_field_csv(filename, **kwargs)
+    elif any(filename.lower().endswith(ext) for ext in _import_field_image_extensions):
```

Comment:
The PIL can read a lot more formats. Instead of identifying the file format ourselfs, why not checking for csv or png and otherwise just try with the PIL?